korge/src/commonMain/kotlin/com/soywiz/korge/ui/UIComboBox.kt

```diff
@@ -62,62 +63,72 @@ open class UIComboBox<T>(
             selectedButton.simulateUp()
             expandButton.simulateUp()
         }
-		invisibleRect.onClick {
-			showItems = !showItems
-			onSizeChanged()
-		}
-	}
+        invisibleRect.onClick {
+            showItems = !showItems
+            onSizeChanged()
+        }
+    }
 
     fun open() {
-        showItems = true
+        addChild(itemsView)
     }
 
     fun close() {
-        showItems = false
+        removeChild(itemsView)
     }
```

Comment:
Maybe, instead of mutating `showItems` variable, could it be an immutable property like `val shownItems get() = itemsView.parent == this` ?

Reply:
@soywiz: Sounds good to me!

Reply:
@soywiz: Sounds good to me!

Reply:
Can you change it in this PR?

Reply:
Hey @soywiz,

I've looked at the code and it turns out the showItems var is necessary. It's used to determine if the dropdown should show instead of if it's visible. We'll have to leave as-is.

BTW, the assignments to showItems in the above code are removed from the PR

J-F